canonical-web-and-design/manager.assets.ubuntu.com
# Improve the asset card layout

Improve the layout of the asset card.

## QA
- Set your .env.local according to the readme.
- ./run
- Make sure the image is now at the top of the card, there is an action button to copy the asset link and a button to open the edit page.
